jsettlers.logic/src/jsettlers/logic/movable/MovableStrategy.java

```diff
@@ -45,10 +46,12 @@
 public abstract class MovableStrategy implements Serializable {
 	private static final long serialVersionUID = 3135655342562634378L;
 
-	private final Movable movable;
+	protected final Movable movable;
+	protected final AbstractMovableGrid grid;
```

Comment:
As it is used so often, its cleaner to keep it and we save some function calls ^^.